pt-BR/mail/chrome/messenger/defaultClientDialog.dtd

```diff
@@ -1,13 +1,13 @@
-<!ENTITY defaultClient.title  "Default Client">
-<!ENTITY defaultClient.intro  "Use &brandShortName; as the default client for:">
+<!ENTITY defaultClient.title  "Cliente padrão">
+<!ENTITY defaultClient.intro  "Usar o &brandShortName; como cliente padrão para:">
 
-<!ENTITY email.label "E-Mail">
-<!ENTITY newsgroups.label "Newsgroups">
-<!ENTITY rss.label "RSS Feeds">
+<!ENTITY email.label "Emails">
+<!ENTITY newsgroups.label "Grupos de notícias">
+<!ENTITY rss.label "Feeds RSS">
 
-<!ENTITY checkOnStartup.label "Always perform this check when starting &brandShortName;">
+<!ENTITY checkOnStartup.label "Sempre verificar ao iniciar o &brandShortName;">
 <!ENTITY checkOnStartup.accesskey "A">
 
-<!ENTITY pb.defaultClient.OSX.desc "Note: &brandShortName; must be set as your default mail client to interoperate with iCal, iPhoto, and Safari. ">
-<!ENTITY pb.defaultClient.OSX.appstore.desc "Note: &brandShortName; must be set as your default mail client to interoperate with Safari. ">
+<!ENTITY pb.defaultClient.OSX.desc "Observação: O &brandShortName; deve ser configurado como o seu cliente de email padrão para interagir com o iCal, o iPhoto e o Safari. ">
```

Comment:
@brunotiussi Understood. After receiving Sherman's approval, either use e-mail or email (chose one to keep consistency).